platforms/ide/tooling-api/build.gradle.kts

```diff
@@ -33,6 +33,7 @@ dependencies {
     implementation(project(":persistent-cache"))
 
     implementation(libs.guava)
+    implementation(libs.jsr305)
```

Comment:
@reinsch82 is this dependency intentional?

Reply:
I introduced it because of the Nullable annotations.
but it seems it is not necessary anymore.